cli/uninstall/uninstall.go

```diff
@@ -133,6 +147,21 @@ func UninstallProgram(program string, programVersion string, binDst string, head
 	return err
 }
 
+// getAllTtVersionFormats returns all version formats with 'v' prefix and
+// without it before x.y.z version.
+func getAllTtVersionFormats(ttVersion string) []string {
+	var versionsToDelete []string
+
+	// Need to determine if we have x.y.z format in tt uninstall argument
+	// to make sure we add version prefix.
+	if strings.Contains(ttVersion, ".") && !strings.HasPrefix(ttVersion, "v") {
```

Comment:
The logic still incompatible with the `install`.

`tt install 123` installs 123 if exist or v123 if exist.
`tt uninstall 123` will not uninstall v123.

You need to add the major.minor.patch check with `regexp.Match("^[0-9]+\.[0-9]+\.[0-9]+", ttVersion)` check to the `tt install` in sync with `tt uninstall` or don't add the confusing (`strings.Contains(ttVersion, ".")`) condition here.

Reply:
Added regular expression version check symmetrically for `tt install` and `tt uninstall`.